src/update-injuries.R

```diff
@@ -13,12 +13,14 @@ cli::cli_alert_info("Fetching schedule...")
 weeks <- nflreadr::load_schedules() |>
   dplyr::mutate(game_type = dplyr::case_when(game_type == 'REG' ~ 'REG',
                                              game_type %in% c('WC','DIV','CON','SB') ~ 'POST',
-                                             T~NA_character_)) |>
+                                             T~NA_character_),
+                game_type = factor(game_type,levels=c('REG','POST')),
+                week = dplyr::case_when(game_type == 'REG' ~ as.integer(week),
+                                        T ~ as.integer(week - max(week[game_type=='REG'])))) |>
   dplyr::filter(season >= most_rec_season) |>
-  dplyr::mutate(week = as.numeric(as.factor(week))) |>
   dplyr::ungroup() |>
   dplyr::group_by(season, game_type, week) |>
-  dplyr::summarise(games_played = sum(!is.na(result))) |>
+  dplyr::summarise(games_played = sum(!is.na(result)),.groups="keep") |>
```

Comment:
Am I reading this correctly? Keep and then ungroup? 😝

Reply:
🤦 i need to stop coding before i get my coffee